FEXCore/Source/Interface/Core/JIT/JITClass.h

```diff
@@ -61,6 +62,15 @@ class Arm64JITCore final : public CPUBackend, public Arm64Emitter {
   const bool HostSupportsRPRES {};
   const bool HostSupportsAFP {};
 
+  struct RestartOptions {
+    FEXCore::LongJump::JumpBuf RestartJump;
+    bool NeedsLongJumps {};
```

Comment:
Overloading the term "LongJump" here to refer to far branches in our JIT is really confusing. Can you find a non-overloaded name and ideally add a comment to clarify what the purpose of the variable is?